internal/controller/verticalpodautoscaler/admission_plugin.go

```diff
@@ -41,5 +53,12 @@ func AdmissionPluginArgs(vpa *v1.VerticalPodAutoscalerController, cfg *Config) [
 		TLSCACertFileArg.Value("/data/tls-ca-certs/service-ca.crt"),
 		WebhookTimeout.Value("10"),
 	}
+	if !CheckArgExists(s.DeploymentOverrides.Admission.Container.Args, KubeAPIQPSArg) {
+		args = append(args, KubeAPIQPSArg.Value("25.0"))
+	}
+	if !CheckArgExists(s.DeploymentOverrides.Admission.Container.Args, KubeAPIBurstArg) {
+		args = append(args, KubeAPIBurstArg.Value("50.0"))
+	}
+
```

Comment:
1. Am I correct in interpreting this as "assert default QPS/Burst centrally via the operator rather than letting the operands set defaults" ? 
2. And I was going to say "should these values be vars/constants somewhere instead of bare values", but `WebhookTimeout.Value("10")` is right there looking at me so I didn't :stuck_out_tongue: 
3. And now I have a 3rd question, why did only the admission plugin get this plumbing and not the recommender or the updater?

Reply:
1. Yeah, I'm trying to set it as a default via the operator rather than through the operands.
2. :stuck_out_tongue: 
3. The defaults for those are already 5/10 for qps/burst and according to the spike/table anyways it should be fine, so I didn't touch them.